cobalt/browser/global_features.cc

```diff
@@ -0,0 +1,130 @@
+// Copyright 2025 The Cobalt Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include "cobalt/browser/global_features.h"
+
+#include "base/no_destructor.h"
+#include "base/path_service.h"
+#include "cobalt/browser/metrics/cobalt_metrics_services_manager_client.h"
+#include "components/metrics/metrics_service.h"
+#include "components/metrics_services_manager/metrics_services_manager.h"
+#include "components/prefs/in_memory_pref_store.h"
+#include "components/prefs/json_pref_store.h"
+#include "components/prefs/pref_service.h"
+#include "components/prefs/pref_service_factory.h"
+
+namespace cobalt {
+
+const char kExperimentConfigFeature[] = "features";
+const char kExperimentConfigFeatureParams[] = "feature_params";
+const char kExperimentConfigExpIds[] = "exp_ids";
+
+constexpr base::FilePath::CharType kExperimentConfigFilename[] =
+    FILE_PATH_LITERAL("Experiment Config");
+
+GlobalFeatures* g_global_features = nullptr;
```

Comment:
This should have the keyword `static`. But looking at the code (e.g. [this search](https://source.chromium.org/search?q=%22GetInstance()%20%22%20f:content%2F%20f:%5C.cc&sq=&ss=chromium)) I think it's more common to have this inside the `GetInstance()` and use either `base::NoDestructor` or `base::Singleton` as a wrapper for the function-level static variable.